src/infrastructure/cloudstorage-connector/cloudstorage.go

```diff
@@ -0,0 +1,47 @@
+package cloudstorage_connector
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/option"
+)
+
+type CloudStorageConnector struct {
+}
+
+type FileData struct {
+	Obj []byte
+}
+
+func (h CloudStorageConnector) GetObject(cloudCredential interface{}, bucketName, objectName string) FileData {
```

Comment:
Aqui seria melhor tipar essa parametro com uma struct, pois isso deixa claro o que precisa ser enviado pro metodo, receber uma interface deixa aberto pra quem for implementar fazer coisas erradas.. :D